# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mesa/X11R7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
Only in Mesa-6.3.2-fixed/configs/: current
diff -ur Mesa-6.3.2/configs/linux-dri Mesa-6.3.2-fixed/configs/linux-dri
--- Mesa-6.3.2/configs/linux-dri	2005-08-20 00:03:05.000000000 +0200
+++ Mesa-6.3.2-fixed/configs/linux-dri	2005-10-09 12:51:55.000000000 +0200
@@ -9,7 +9,7 @@
 CC = gcc
 CXX = g++
 
-MKDEP = /usr/X11R6/bin/makedepend
+MKDEP = /usr/X11/bin/makedepend
 #MKDEP = gcc -M
 #MKDEP_OPTIONS = -MF depend
 
@@ -25,7 +25,7 @@
 	-DPTHREADS -DUSE_EXTERNAL_DXTN_LIB=1 -DIN_DRI_DRIVER \
 	-DGLX_DIRECT_RENDERING -DHAVE_ALIAS
 
-X11_INCLUDES = -I/usr/X11R6/include
+X11_INCLUDES = -I/usr/X11/include
 
 CFLAGS   = $(WARN_FLAGS) $(OPT_FLAGS) $(PIC_FLAGS) $(ARCH_FLAGS) $(DEFINES) \
 	$(ASM_FLAGS) -std=c99 -ffast-math 
@@ -36,7 +36,7 @@
 ASM_SOURCES = 
 
 # Library/program dependencies
-EXTRA_LIB_PATH=-L/usr/X11R6/lib
+EXTRA_LIB_PATH=-L/usr/X11/lib
 
 LIBDRM_LIB = `pkg-config --libs libdrm` -ldrm
 DRI_LIB_DEPS  = $(EXTRA_LIB_PATH) -lm -lpthread -lexpat -ldl $(LIBDRM_LIB)
